### ISSUE TITLE
Short timelimit at the first execution 

### ISSUE THREAD

**Issue by smbct** [aritrasep/FLLH.jl#1]
Using a short time limit at the first execution of the algorithm may cause an error. Indeed julia does not have enough time to compile each package.

To fix this issue, it is possible to execute an initial code (see  "Compiling Small Instances:" [here](https://aritrasep.github.io/FPBH.jl/docs/build/getting_started.html)) or to use a longer timelimit (120 seconds may work, depending on the machine).

**Comment by aritrasep** [comment]
Yup. I have thought of resolving this issue in various ways. Currently, the only solution for this issue is to execute the initial code (mentioned under [Compiling Small Instances](https://aritrasep.github.io/FPBH.jl/docs/build/getting_started.html)) during the first run or whenever you switch the lp solver during the session.
